src/istgtcontrol.c

```diff
@@ -977,6 +977,56 @@ exec_replica(UCTL_Ptr uctl)
 	}
 	return (UCTL_CMD_OK);
 }
+
+static int
+exec_max_io_wait(UCTL_Ptr uctl)
+{
+	const char *delim = ARGS_DELIM;
+	char *arg;
+	int rc = 0;
+	char *s_io_wait_time;
+	char *result;
+
+	if (uctl->setargcnt >= 1)
+		s_io_wait_time = uctl->setargv[0];
+	else
+		s_io_wait_time = NULL;
+
+	if (s_io_wait_time)
+		uctl_snprintf(uctl, "%s \"%s\" \n", uctl->cmd, s_io_wait_time);
+	else
+		uctl_snprintf(uctl, "%s\n", uctl->cmd);
+
+	rc = uctl_writeline(uctl);
+	if (rc != UCTL_CMD_OK) {
+		return (rc);
+	}
+
+	/* receive result */
+	while (1) {
+		rc = uctl_readline(uctl);
+		if (rc != UCTL_CMD_OK) {
+			return (rc);
+		}
+		arg = trim_string(uctl->recvbuf);
+		result = strsepq(&arg, delim);
+		strupr(result);
+		if (strcmp(result, uctl->cmd) != 0)
+			break;
+		if (uctl->iqn != NULL) {
```

Comment:
why this 'iqn' check?

Reply:
This check is unnecessary. I will remove this.